src/axes/numericAxis.ts

```diff
@@ -262,6 +278,26 @@ export module Axes {
       }
     }
 
+    /**
+     * Gets the approximate text width setting.
+     *
+     * @returns {boolean} The current approximate text width setting.
+     */
+    public approximateTextWidth(): boolean;
```

Comment:
I'm not a fan of the endpoint name.  If I were to call

`numericAxis.approximateTextWidth()`, what would I expect to be returned?

There's probably a better name, but maybe `useTextWidthApproximation()`?